index.d.ts

```diff
@@ -246,8 +246,28 @@ interface PrefixType {
     showSlash?: boolean;
 }
 
-type ImageSizeType = 'small' | 'medium' | 'large';
-type CardCoverageType = '25%' | '33%' | '50%' | '67%' | '75%';
+type CardImageType = {
+    element: ImageBlockElement;
+    position?: 'left' | 'top' | 'right';
+    size?: ImageSizeType;
+};
+
+type SmallHeadlineSize = 'tiny' | 'xxsmall' | 'xsmall';
+
+interface CardType {
+    linkTo: string;
+    pillar: Pillar;
+    headlineString: string;
+    headlineSize?: SmallHeadlineSize;
+    webPublicationDate?: string;
+    prefix?: PrefixType;
+    image?: CardImageType;
+    standfirst?: string;
+    percentage?: CardPercentageType;
```

Comment:
Would `width` better convey the intention of this prop than `percentage`?

Reply:
`width?: CardWidthType;` I like it.

You could perhaps go further and do:

`type CardWidthType = 'slim' | 'medium' | 'generous';` (but with _much_ better names than those).

Maybe... On the _other_ hand, the problem here is these percentages need to add up so if you abstract this too much the api could become hard to use?

Reply:
Hmm, I'm actually now thinking keeping `percentage` might better inform the user about how they need to use this prop.